asmcomp/arm64/emit.mlp

```diff
@@ -1009,6 +1032,30 @@ let rec emit_all env i =
 
 (* Emission of a function declaration *)
 
+type preproc_fun_result =
+  { max_stack_size : int;
```

Comment:
`s/max_stack_size/max_frame_size`? Really, this function is computing the maximum value of `env.stack_offset`. Add a comment?

Also affects `amd64/emit.mlp`. Perhaps we should move this function to `asmcomp/emitaux.ml` instead of duplicating this for every backend.

Reply:
Yes, moving shared parts to emitaux.ml sounds good.  Eventually, I'd like to refactor these computations of various aspects of the stack frame that are currently spread between Selection and Emit, but I'll need more time.

Reply:
Done; both the rename and factoring out the cut-and-paste into emitaux.ml

Reply:
Thanks.